packages/grid/backend/grid/core/config.py

```diff
@@ -138,7 +138,7 @@ def get_emails_enabled(self) -> Self:
         True if os.getenv("CREATE_PRODUCER", "false").lower() == "true" else False
     )
     N_CONSUMERS: int = int(os.getenv("N_CONSUMERS", 1))
-    SQLITE_PATH: str = os.path.expandvars("$HOME/data/db/")
+    SQLITE_PATH: str = os.path.expandvars("/tmp/syft/")
```

Comment:
Hi @shubham3121 I changed this to `/tmp/syft` to solve sqlalchemy can't open database file in `/root/data/db`, but not sure it's optimal. Also in `postgres-statefulset.yaml`  the `mountPath` is set to `/data/db`, should be change it to be the same to value of `SQLITE_PATH`?

Reply:
What error do we get with `/root/data/db` ? Does it complain about directory doesn't exists or some sort of permission error ? Earlier we used to dir if doesn't exist, I think sqlalchemy could do that same maybe via some config.

Reply:
Its seems like a permission issue to create a directory at the root level. Maybe we can create the director beforehand (while changing ownership of directory to user)

Reply:
but is there any probably with using `/tmp/syft` since we use it in the local python version?

Reply:
after discussion changed it to `/tmp/data/db` for both single container and k8s mode in this commit [cc8e976](https://github.com/OpenMined/PySyft/pull/9176/commits/cc8e9762d75e300ce522daf007b40db65c4eacee)